src/main/java/Ui.java

```diff
@@ -9,59 +9,91 @@
      */
     Scanner scanner = new Scanner(System.in);
 
+    /**
+     * Displays a message indicating an error in loading the task list save file.
+     */
     public void showLoadingError() {
         String message = "Cannot find save file.\n"
                 + "Creating a new one, even if it's a bother";
 
         say(message);
     }
 
+    /**
+     * Displays the list of tasks given.
+     * @param tasks the list of tasks
+     */
     public void showTasks(TaskList tasks) {
         String message = "Here are your procrastinated tasks:\n"
                 + tasks;
 
         say(message);
     }
 
-    public void showDeletedTask(Task t) {
-        String message = "Giving up already? Removed this:\n\t" + t;
+    /**
+     * Displays the deleted task.
+     * @param task the deleted task
+     */
+    public void showDeletedTask(Task task) {
+        String message = "Giving up already? Removed this:\n\t" + task;
 
         say(message);
     }
 
-    public void showDonetask(Task t) {
-        String message = "Yay. You've finally done this task:\n\t" + t;
+    /**
+     * Displays the task marked as completed.
+     * @param task the completed task
+     */
+    public void showDonetask(Task task) {
+        String message = "Yay. You've finally done this task:\n\t" + task;
 
         say(message);
     }
 
-    public void showAddedTask(Task t, int count) {
+    /**
+     * Displays the newly added task and the total number of tasks.
+     * @param task the added task
+     * @param count the total number of tasks
+     */
+    public void showAddedTask(Task task, int count) {
         String message = "Another task? Oh well, here's the task:\n"
-                + "\t" + t + "\n"
```

Comment:
Perhaps, you can consider using System.lineSeparator() as a replacement for "\n" for system compatibilities.